cms/djangoapps/contentstore/tasks.py

```diff
@@ -70,6 +71,11 @@
 LOGGER = get_task_logger(__name__)
 FILE_READ_CHUNK = 1024  # bytes
 FULL_COURSE_REINDEX_THRESHOLD = 1
+ALL_ALLOWED_XBLOCKS = frozenset(
+    [entry_point.name for entry_point in pkg_resources.iter_entry_points("xblock.v1")]
+    +
+    ['wiki']
```

Comment:
add a comment about why `wiki` is being added here for future reference other then that, looks good 👍

Reply:
Adding wiki here b/c it is not an xblock but we would like to support it.